src/components/Contacts/ContactDetails/ContactDetailsTab/PartnerAccounts/ContactDetailsPartnerAccounts.tsx

```diff
@@ -37,19 +37,33 @@ const ContactPartnerAccountsTextContainer = styled(Box)(({ theme }) => ({
   margin: theme.spacing(2),
   alignItems: 'center',
 }));
+const PartnerAccountTextContainer = styled(Box)(() => ({
+  display: 'flex',
+  alignItems: 'center',
+}));
 
+const PartnerAccountTextLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.secondary,
+  marginRight: '5px',
+  fontSize: '13px',
+}));
 interface ContactDetailsPartnerAccountsProps {
-  contact: ContactPartnerAccountsFragment;
+  accountListId: string;
+  contactId: string;
 }
 
 export const ContactDetailsPartnerAccounts: React.FC<
   ContactDetailsPartnerAccountsProps
-> = ({ contact }) => {
+> = ({ accountListId, contactId }) => {
+  const { data } = useContactDonorAccountsQuery({
+    variables: { accountListId, contactId },
+    fetchPolicy: 'no-cache',
```

Comment:
Do you mind adding a comment here to mention that the API is returning different results from `displayName` depending on how we query it, so to workaround we're not caching the results of this query to avoid poluting the Apollo cache?